src/constants/config.json

```diff
@@ -6,8 +6,9 @@
   },
   "eduAPI": {
     "authToken": "64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188",
-    "classrooms": "teachers/classrooms.json",
-    "root": "https://education.staging.zooniverse.org/"
+    "root": "https://education.staging.zooniverse.org/",
+    "students": "/students/classrooms/",
+    "teachers": "teachers/classrooms/"
```

Comment:
Query: is there a reason that `/students/classrooms/` starts with a `/` but `teachers/classrooms/` doesn't?